app/views/layouts/engine_syndicated.html.erb

```diff
@@ -15,10 +15,12 @@
     </div>
   </div>
   <% content_for(:javascripts) do %>
-    <%= javascript_include_tag "syndication/iframeResizer" %>
-    <script>
-      window.iframeResizer('MASRESIZE-').start();
-    </script>
+    <% unless exclude_syndicated_iframe_resizer? %>
```

Comment:
My only suggestion would be to flip the logic and the name....for example
if include_syndicated_iframe_resizer?

other than that :+1: